src/python/doc/rips_complex_user.rst

```diff
@@ -54,7 +54,7 @@ regular Rips complex.
 
 Another option which is especially useful if you want to compute persistent homology in "high" dimension (2 or more,
 sometimes even 1), is to build the Rips complex only up to dimension 1 (a graph), then use
-:func:`~gudhi.SimplexTree.collapse_edges` to reduce the size of this graph, and finally call
+:func:`~gudhi.SimplexTree.flag_filtration.edge_collapse.reduce_graph` to reduce the size of this graph, and finally call
```

Comment:
```suggestion
:func:`~gudhi.flag_filtration.edge_collapse.reduce_graph` to reduce the size of this graph, and finally call
```

Reply:
Oups, I messed up those 2 in my search&replace... I should probably leave them as they were, or rephrase a bit if I want to point to the new function.